➕ Install eslint

fgottesmann opened the pull request:
https://eslint.org/docs/user-guide/getting-started
https://github.com/prettier/eslint-config-prettier
Commands for installation and initialisation:
`npm install eslint --save-dev`
`npx eslint --init`

fgottesmann committed:
➕ Install eslint
https://eslint.org/docs/user-guide/getting-started
`npm install eslint --save-dev`
`npx eslint --init`

fgottesmann committed:
⬆️ Format eslintrc.json

fre-ben reviewed (approved):
LGTM!
You could add the link to ESLint/Prettier config though: https://github.com/prettier/eslint-config-prettier

fgottesmann committed:
➕ Install eslint
https://eslint.org/docs/user-guide/getting-started
`npm install eslint --save-dev`
`npx eslint --init`
-Add nextjs typings [https://nextjs.org/docs/basic-features/typescript]
🔨 Add eslint script in package.json
